spring-kafka/src/main/java/org/springframework/kafka/listener/ContainerProperties.java

```diff
@@ -610,6 +642,21 @@ public void setSubBatchPerPartition(boolean subBatchPerPartition) {
 		this.subBatchPerPartition = subBatchPerPartition;
 	}
 
+	public AssignmentCommitOption getAssignmentCommitOption() {
+		return this.assignmentCommitOption;
+	}
+
+	/**
+	 * Set the assignment commit option. Default {@link AssignmentCommitOption#ALWAYS}.
+	 * In a future release it will default to {@link AssignmentCommitOption#LATEST_ONLY}.
```

Comment:
I think we need an issue for this do not forget to really change it in the "future release"